### ISSUE TITLE
Content of yellow-system.ini has been modified

### ISSUE THREAD

**Issue by raphraph** [YunoHost-Apps/yellow_ynh#29]
### The bug

My website was showing an error stating that the "default theme does not exist" and therefore it was not rendering anything (no text, no images, only the title/links of the different pages of the website).

I found out in the site settings that the site name, author name, webmaster contact etc had been reset to default values, so I checked the content of the file `yellow-system.ini` since this file contains this information.

The only content of the file was : `UpdateEventDaily: 1662076843`

So I modified the file correspondingly to what it is supposed to contain :
```
# Datenstrom Yellow system settings

Sitename: Datenstrom Yellow
Author: Datenstrom
Email: webmaster
Layout: default
Theme: stockholm
Language: en
Parser: markdown
Status: public
```
and the website was rendered correctly again.

I don't know why and how the content of `yellow-system.ini` was changed... System logs do not show any updates or changes...

### Context

- Hardware: *Raspberry Pi at home*
- YunoHost version: 4.4.2.10 (stable)
- I have access to my server: *Through SSH | through the webadmin*
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: *no*

### Steps to reproduce

I dont' know...

### Expected behavior

That the file content does not change unless a system admin changes it.

### Logs

This is the content of `yellow-website.log`, it shows what I did last time I realized my website content has disappeared (see issue #26), but it seems nothing happened since :
```
2022-08-25 09:16:45 info Add user 'username'
2022-08-25 13:51:33 error Can't parse file 'system/extensions/core.php'!
2022-08-25 13:51:40 error Can't parse file 'system/extensions/core.php'!
2022-08-25 14:10:03 info Update extension 'Command 0.8.41'
2022-08-25 14:10:03 info Update extension 'Core 0.8.89'
2022-08-25 14:10:03 info Update extension 'Edit 0.8.63'
2022-08-25 14:10:03 info Update extension 'Markdown 0.8.22'
2022-08-25 14:10:03 info Update extension 'Stockholm 0.8.13'
2022-08-25 14:10:04 info Update extension 'Update 0.8.80'
```
Tell me if other logs can be useful.

**Comment by ericgaspar** [comment]
Can you tell me what is the path of this file `yellow-system.ini`?

**Comment by ericgaspar** [comment]
should be fix with #31
